src/controllers/dashboardgeneral.js

```diff
@@ -2,50 +2,126 @@ define(["jQuery", "loading", "fnchecked", "emby-checkbox", "emby-textarea", "emb
     "use strict";
 
     function loadPage(page, config, languageOptions, systemInfo) {
-        systemInfo.CanLaunchWebBrowser ? page.querySelector("#fldRunWebAppAtStartup").classList.remove("hide") : page.querySelector("#fldRunWebAppAtStartup").classList.add("hide"), page.querySelector("#txtCachePath").value = config.CachePath || "", $("#selectLocalizationLanguage", page).html(languageOptions.map(function(l) {
-            return '<option value="' + l.Value + '">' + l.Name + "</option>"
-        })).val(config.UICulture), currentLanguage = config.UICulture, systemInfo.CanSelfUpdate ? page.querySelector(".fldAutomaticUpdates").classList.remove("hide") : page.querySelector(".fldAutomaticUpdates").classList.add("hide"), $("#chkEnableAutomaticServerUpdates", page).checked(config.EnableAutoUpdate), $("#chkEnableAutomaticRestart", page).checked(config.EnableAutomaticRestart), systemInfo.CanSelfRestart ? page.querySelector("#fldEnableAutomaticRestart").classList.remove("hide") : page.querySelector("#fldEnableAutomaticRestart").classList.add("hide"), systemInfo.CanSelfRestart || systemInfo.CanSelfUpdate ? $(".autoUpdatesContainer", page).removeClass("hide") : $(".autoUpdatesContainer", page).addClass("hide"), loading.hide()
+        if (systemInfo.CanLaunchWebBrowser) {
+            page.querySelector("#fldRunWebAppAtStartup").classList.remove("hide");
+        } else {
+            page.querySelector("#fldRunWebAppAtStartup").classList.add("hide");
+        }
+        page.querySelector("#txtCachePath").value = config.CachePath || "";
```

Comment:
this seems to be doing the same as #205 does, you might want to coordinate with @dkanada

Reply:
When i rebased, i assumed all changes made in #205 as the most recent.